oak/server/rust/oak_runtime/src/node/mod.rs

```diff
@@ -93,6 +93,7 @@ pub fn create_node(
     application_configuration: &ApplicationConfiguration,
     node_configuration: &NodeConfiguration,
     grpc_configuration: &GrpcConfiguration,
+    signature_configuration: &HashMap<String, SignatureConfiguration>,
```

Comment:
I think the term "configuration" is already overloaded, how about signature database / repository / list / map / table or something like that?